meson.build

```diff
@@ -6,7 +6,7 @@ project(
   # tools/version_utils.py
   version: '1.9.0.dev0',
   license: 'BSD-3',
-  meson_version: '>= 0.60',
+  meson_version: '>= 0.60,< 0.70.0',
```

Comment:
This is a little arbitrary, but we should probably set it a little tighter. Latest released version is `0.62.2`, and we need to get a bunch of features that will help into 0.63-0.65. So I'd take a guess saying `<0.66` is about right.